### ISSUE TITLE
Headings are inconsistent (and increasingly unhelpful) throughout the app

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#3860]
## What happened?

Headings matter for screen reader users. This quote describes [why headings matter on web pages](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/), and the principles also apply to mobile: 

> Headings are the backbone of the content of a page. A visitor should be able to scan the webpage using headings to get a good impression of its content. Heading levels have meaning, especially for screen reader users .... headings [represent] the page content similar to the index of a book: easily scannable and semantic.

Right now, our heading use is not consistent throughout the app, and many of the new features we're working on have few (if any) headings. There's not a great metaphor for sighted folks to understand exactly how that impacts screen reader users, but the closest thing I can think of is: 

Pretend that the only thing you could see when you landed on a page were the headings. Could you figure out 1) what information was on the page (_is the thing I'm looking for on this page?_) and 2) the fastest way to get to that information (_to find X, I will look at the stuff under Y heading_)

I argue that right now, on most of our details pages, you can't really do the stuff above well. [Here's a set of slides where you can test navigating via headings for yourself](https://docs.google.com/presentation/d/1Ei1-VsVyPAuuF0Jf3YAzDk7FqUFnULxh5vuO3-XaHr8/edit?usp=sharing).

The above slides are just a VERY cursory look at our headings. Ideally we will:
- Come up with clear guidelines for what should have a heading
- Do a full audit of pages in the app and how they're currently using headings
- Write tickets for what changes should be made on which pages
- Make the changes in those tickets

This particular ticket can be closed after the work is done for the first three bullet points: come up with guidelines, audit the app, and document specific shortcomings to be addressed.

**Comment by bischoffa** [comment]
Changing this from a bug ticket to an enhancement ticket. This seems much larger than a bug and also led to a separate epic that is needed to handle this work. Per the epic this ticket is blocked by [4972](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/4972)

**Comment by bischoffa** [comment]
Blocked by ticket 4972 which was created back in March Spacing WCAG 2.2. I do not expect any ETA or timing on this work getting prioritized so I moved it to icebox.

**Comment by TKDickson** [comment]
Auditing is already finished - see many tickets in the linked epic. New epic created for making the changes: #7817. Closing.
